src/testHelpers/setupTests.js

```diff
@@ -17,6 +17,20 @@ const reactErrorsLimitPerTestSuite = 6; // The goal is to have zero React errors
 
 // eslint-disable-next-line no-console
 console.error = (message, ...rest) => {
+  const { expectedWarnings } = window;
+  if (expectedWarnings && typeof expectedWarnings === 'object') {
+    const warningsRegex = new RegExp(
+      [REACT_FAILED_PROP_TYPE, ...Object.values(expectedWarnings)].join('.*'),
+    );
+    if (warningsRegex.test(message)) {
```

Comment:
this currently doesnt allow you to supress 2 prop warnings at a time. E.g. below `ImageContainer` has 2 required props, `blocks` and `foobar`
```
describe('with no data', () => {
    suppressPropWarnings(['blocks', 'ImageContainer']);
    suppressPropWarnings(['foobar', 'ImageContainer']);
    isNull('should return null', <ImageContainer />);
  });
```

It will only supress 1 of the 2 console errors

Reply:
We wouldn't be able to test for that because once an error is thrown the rest of the code isn't executed. In cases where multiple props absences are detected, only one console error will be thrown so we can catch them all like this:

```
describe('with no data', () => {
    suppressPropWarnings(['blocks', 'foo', 'bar', 'ImageContainer']);
    isNull('should return null', <ImageContainer />);
  });
```

Reply:
>once an error is thrown the rest of the code isn't executed

Yeah it is, you can have 2 prop warning errors happen in 1 component.

Reply:
![image](https://user-images.githubusercontent.com/11341355/63090751-29259e00-bf54-11e9-9d76-a594739b65d3.png)

is from
```
describe('Image', () => {
  describe('with no data', () => {
    suppressPropWarnings(['blocks', 'array']);
    isNull('should return null', <ImageContainer />);
  });
});
```
when ImageContainer has
```
ImageContainer.propTypes = {
  foobar: string.isRequired,
  barfoo: string.isRequired,
};
```